### ISSUE TITLE
EDC - MXD - Wallet Upgrade

### ISSUE THREAD

**Issue by stefan-ettl** [eclipse-tractusx/sig-release#613]
As a developer I want to use the MXD to get familiar with the dataspace technology Catena-X uses. So that I can build up enough knowledge to prepare my own Systems and data to be shared in a dataspace.

Within PI13, the MIW is also getting upgraded to support the IATP (Identity and Trust Protocol). This upgraded MIW needs to be integraded to the MXD.

- [ ]  The upgraded MIW is integrated in the MXD and can be used.

**Comment by stephanbcbauer** [comment]
was presented in open planning ⇾ dependencies are identified, developer and committer are involved.

**Comment by stephanbcbauer** [comment]
Hello @stefan-ettl , @hemantxpatel, @lgblaumeiser 

Since the feature is a 24.08 feature and the development phase is coming to an end, we need a status on the feature.

- [ ] Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
- [ ] Please check whether the status (backlog, work in progress ...) is set correctly
- [ ] Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
- [ ] Please comment on the current status of the feature
- [ ] Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.

If you need any clarification, please get in touch, thank you very much.
Stephan

**Comment by lgblaumeiser** [comment]
2024-06-11: Current state:
- There is an open Pull Request on MIW side, so that it is usable in the MXD
- Open Question: For EDC to work with MIW as wallet, there is a need for a custom EDC. This is setup, the question is, how the EDC is published for use in MXD. The result of the discussion is, that it is preferable to provide the EDC as buildable image instead of publishing it in DockerHub, as the support efforts for a published image are substantial and the MXD works also without a published image.
- As soon, as MIW PR is through, work will progress on the finalization of the MXD with MIW wallet.

**Comment by lgblaumeiser** [comment]
2024-06-11:

* [x]  Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
* [x]  Please check whether the status (backlog, work in progress ...) is set correctly
* [x]  Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
* [x]  Please comment on the current status of the feature, see comment above
* [ ] Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.

**Comment by hemantxpatel** [comment]
Related Issues:
- eclipse-tractusx/tutorial-resources#349
- eclipse-tractusx/tutorial-resources#350
- eclipse-tractusx/tutorial-resources#351

**Comment by lgblaumeiser** [comment]
Current state: Topic is blocked by MIW changes that needs to be finished. Risk, that this is not finished for the release

**Comment by stephanbcbauer** [comment]
Hello @lgblaumeiser, @hemantxpatel 
 
In a few weeks, we want to publish Tractus-X Release 24.08. For this, we need a clear status about the features (which will be included in the release and which will be cancelled or postponed to the next release).
 
Currently, the feature is still in status `Work in progress`. Please update the status by 09.07.24 eob.
 
If the feature is not ready for release 24.08, or it is possibly not planned at all, please set the status to `Not in release`. Please remember that in this case, any dependent components must be informed.
 
Thank you and best regards.

**Comment by hemantxpatel** [comment]
Hi @stephanbcbauer, This issue is blocked due to MIW changes (IATP implementation) not yet complete, Once they are done, then only we will be able to integrate MIW into MXD.

**Comment by lgblaumeiser** [comment]
As discussed today, this might be of interest: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub, content is currently under ip clearance, but an initial contributor is there who can be contacted.

**Comment by lgblaumeiser** [comment]
Hello, @hemantxpatel, as you did not participate in the weekly lately, I would like to ask about the status of this feature. Do you still work on MXD or did you stop working on it. Do you know about someone currently working on MXD?

**Comment by lgblaumeiser** [comment]
There is an adaptation of the MXD available which makes use of the Identity Hub as wallet. The setup is available in the [tutorial resources repository](https://github.com/eclipse-tractusx/tutorial-resources). I close this issue, although the documentation is not perfect.
